lib/instrumentation/modules/knex.js

```diff
@@ -7,7 +7,7 @@ var symbols = require('../../symbols')
 
 module.exports = function (Knex, agent, { version, enabled }) {
   if (!enabled) return Knex
-  if (semver.gte(version, '0.22.0')) {
+  if (semver.gte(version, '0.95.16')) {
```

Comment:
```suggestion
  if (semver.gte(version, '1.0.0')) {
```

How about making the guard on `<1.0.0` on the assumption that there will likely never be an 0.96.x release, but if there is, it will be compatible?